src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -167,6 +167,18 @@ public static Calendar convertToUserTimeZone(Calendar time, double timeZone) {
     }
 
     /**
+     * Converts the {@code localDate} to UTC by adding/subtracting the {@code localTimeZone} offset.
+     * 
+     * @deprecated Method should be removed once all time data is migrated to UTC. 
+     */
+    @Deprecated
+    public static Date convertLocalDateToUtc(Date localDate, double localTimeZone) {
+        Calendar localCal = dateToCalendar(localDate);
+        localCal.add(Calendar.MILLISECOND, (int) (60 * 60 * 1000 * (-localTimeZone)));
```

Comment:
Just to confirm, if I were to use Calendar.MINUTE then line 177 would look like this:

```JAVA
localCal.add(Calendar.MINUTE, (int) (60 * 1000 * (-localTimeZone)))
```

Is that correct? If so, then why is this better than the `MILLISECOND` alternative?